www/front_src/src/styles/partials/_side-nav.scss

```diff
@@ -13,6 +13,7 @@
   text-align: center;
   background: #e0e0e0;
   transition: all 0.2s;
+  z-index: 2;

```

Comment:
modification from another PR not yet validated by the QA -> to remove from this PR

Reply:
this one is necessary for this feature too